includes/wikia/GlobalTitle.php

```diff
@@ -835,6 +844,32 @@ private function loadNamespaceNames() {
 	}
 
 	/**
+	 * Get $wgExtraNamespaces for title language
+	 *
+	 * @return array
+	 */
+	private function getLocalWgExtraNamespaces() {
```

Comment:
`wgExtraNamespacesLocal` is fetched in correct language from wikifactory: https://github.com/Wikia/app/pull/11501/files#diff-ff18afd0f5f75f543e37ee23266e47f5R840

I'm not sure how would you like to handle it better.